src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -118,6 +119,11 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
         anon_user_info_emoji_name: item?.anon_user_info_emoji_name
       }}
       postNotificationIsMediaOnly={item?.rawJson?.isMediaOnlyMessage}
+      containerBackgroundColor={
+        item?.channelType === 'POST_NOTIFICATION'
+          ? COLORS.signedPostNotificationChannel
+          : COLORS.anonPostNotificationChannel
+      }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic for setting the `containerBackgroundColor` prop based on the `channelType` seems correct. It checks if the `channelType` is 'POST_NOTIFICATION' and sets the color accordingly.
```diff
+       containerBackgroundColor={
+         item?.channelType === 'POST_NOTIFICATION'
+           ? COLORS.signedPostNotificationChannel
+           : COLORS.anonPostNotificationChannel
+       }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->